langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -346,6 +489,15 @@ public isolated function 'map(xml<ItemType> x, @isolatedParam function(ItemType
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml bookSet = xml `<book><title>Java</title></book><book><title>Ballerina</title></book>`;
+# xml bookTitles = xml ``;
+# 
+# bookSet.forEach(function (xml xmlItem) {
+#   bookTitles += xmlItem.children();
+# });
```

Comment:
Doesn't this produce the same value as that assigned to `bookSet`? Shall we think of a different example?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/7b4dfa608b04064393e46211524556d9850dfddf